packages/lambdas/src/fhir-to-medical-record.ts

```diff
@@ -417,11 +418,10 @@ async function createFeedbackForBrief({
     const extra = { cxId, patientId, aiBriefId: aiBrief.id };
     const { log } = out("createFeedbackForBrief");
     log(`${msg} - error: ${errorToString(error)}, extra: ${JSON.stringify(extra)}`);
-    capture.error(msg, {
-      extra: {
-        ...extra,
-        error,
-      },
+    capture.setExtra({
+      ...extra,
+      error,
     });
+    throw new MetriportError(msg, error);
```

Comment:
Note this was not throwing an error, so we prob don't want to touch this behavior

Reply:
wow good catch thank you, just assumed that all places where we were logging an error should instead throw. But i guess that means more manual testing.